randovania/layout/preset_describer.py

```diff
@@ -131,16 +131,15 @@ def format_params(self, configuration: BaseConfiguration) -> dict[str, list[str]
         starting_locations = configuration.starting_location.locations
         if len(starting_locations) == 1:
             area = game_description.world_list.area_by_area_location(starting_locations[0])
-            starting_location = game_description.world_list.area_name(area)
+            starting_location = f"Starts at {game_description.world_list.area_name(area)}"
```

Comment:
we should probably have this support starting nodes in the future

Reply:
Oh. Didn't even notice. hm